src/GetArticle.jsx

```diff
@@ -13,23 +17,58 @@ export default function GetArticle() {
       })
       .then((result) => {
         setArticleItem(result.article[0]);
+        setIsLoading(false);
       });
   }, []);
 
-  return (
-    <div key={articleItem.article_id}>
-      <h1>{articleItem.title}</h1>
-      <div className="article-header-holder">
-        <p>Article by: {articleItem.author}</p>
-        <p>
-          Votes: {articleItem.votes}
-          <button>UPVOTE TO GO HERE</button>
-        </p>
+  function handleUpvote(event, increment = 1) {
+    event.preventDefault();
```

Comment:
Generally we don't need to worry about preventing the default behaviour of a click, that would usually be for a submit function.

We could make this two functions a bit more DRY by combining them into one if we passed a value from the onClick if you wanted to tidy it up a little more.